website/content/en/docs/olm-integration/quickstart-package-manifests.md

```diff
@@ -4,6 +4,9 @@ linkTitle: Package Manifests Quickstart
 weight: 2
 ---
 
+**Note**
+As operator framework has moved to using bundle format by default, the package manifest commands have been deprecated and will be removed soon. It is suggested that you follow the bundle quickstart][quickstart-bundle] to package your operator. 
```

Comment:
Haven't removed the package manifest Quickstart from the docs yet, but added a note instead. Do we remove it now or in subsequent release?

Reply:
In v2.0.0. Can you put a `<!-- TODO(2.0.0): remove this document -->` comment at the top of the page?